tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from utils.singleton import destroyAllSingletonInstances
+from utils.singleton import noSingletonsAround
+
+@pytest.fixture(scope="function", autouse=True)
+def my_fixture(): # pylint: disable=invalid-name
```

Comment:
`robotTestFixture` may be a better name here - In general, the test environment should grow to have this as the common fixture and inherit in test-specific fixtures